tests/output/InteractionsTest.py

```diff
@@ -4,19 +4,29 @@
 
 from coalib.output.Interactions import fail_acquire_settings
 from coalib.output.printers.LogPrinter import LogPrinter
+from testfixtures import LogCapture
 from coalib.settings.Section import Section
 
 
 class InteractionsTest(unittest.TestCase):
 
     def test_(self):
         log_printer = LogPrinter(NullPrinter())
-        section = Section('')
-        self.assertRaises(TypeError, fail_acquire_settings, log_printer, None,
-                          section)
+        self.assertRaises(TypeError, fail_acquire_settings, log_printer, None)
         self.assertRaises(AssertionError,
                           fail_acquire_settings,
                           log_printer,
-                          {'setting': ['description', 'bear']}, section)
-        self.assertEqual(fail_acquire_settings(log_printer, {}, section), None,
-                         section)
```

Comment:
Are you sure not to remove the `section` parameter with a separate commit ? Up to you :D)